### ISSUE TITLE
AAE doesn't converge

### ISSUE THREAD

**Issue by eugeniaring** [CY-Jeong/anomaly-detection-mvtec#1]
I cloned your repository and I tried to train the aae model with the same commands you suggested. After 100 epochs, the losses began to become nan. Can you provide a training setup that can avoid this problem? Thank you very much.

**Comment by CY-Jeong** [comment]
As you mentioned, depends on learning rate, epochs, the losses become nan, or become accurate value but is not to make fixed result with fixed hyper params. For example, you can get variable returns that might be accurate value, or be nan with fixed epochs and learning rate. So I suggest you to do several times to get good result with chaning hyper params.
Sorry for not clear answer.
